tables.sql

```diff
@@ -10,39 +10,38 @@ CREATE TABLE "items" (
-("Bikes","recumbent",2995.00, "Schwinn", "Schwinn.PNG", "4.5"),
-("Bikes","tandem",2499.99, "Salsa", "SalsaC.PNG", "4.5"),
-("Camping","tent - 1 person",337.39, "Big Agnes", "BigAgnes.PNG", "4.1"),
-("Camping","tent - 2 person",185.00, "TheNorthFace", "TheNorthFace.PNG", "4.5"),
-("Camping","tent - 3 person",179.73, "Klymit", "Klymit.PNG", "5.0"),
-("Camping","tent - 4 person",188.89, "Eureka", "Eureka.PNG", "4.0"),
-("Camping","tent - 6 person",117.73, "Alpine Mountain Gear", "AlpineMountainGear.PNG", "4.0"),
-("Camping","bear canister - S",76.95, "BearVault", "BearVaultA.PNG", "4.2"),
-("Camping","bear canister - M",83.95, "BearVault", "BearVaultB.PNG", "4.0"),
-("Camping","bear canister - L",89.95, "BearVault", "BearVaultC.PNG", "4.5"),
-("Camping","bear canister - XL",94.95, "BearVault", "BearVaultD.PNG", "4.0"),
-("Camping","daypack",65.00, "Osprey", "Osprey.PNG", "4.6"),
-("Camping","overnight pack",83.39, "Co-op Traverse", "Co-opTraverse.PNG", "4.5"),
-("Camping","backpacking pack",239.95, "Granite Gear", "GraniteGear.PNG", "4.5"),
-("Water sports","canoe",999.95, "Old Town", "OldTown.PNG", "4.0"),
-("Water sports","kayak - single",1299.00, "EddyLine", "EddyLine.PNG", "4.5"),
-("Water sports","kayak - tandem",599.95, "TAHE", "TAHE.PNG", "4.0");
+("Snowshoes","mountain terrain",17.00, "Atlas", "atlas.png"),
+("Snowshoes","rolling terrain",13.00, "komperdell", "komperdell.png"),
+("Snowshoes","flat terrain",6.00, "Tubbs", "tubs.png"),
+("Skis","cross country",7.00, "Fischer", "fischer.png"),
+("Skis","downhill",44.00, "Rossignol", "rossignol.png"),
+("Snow tube","snow tube",15.00, "Sonic", "sonic.png"),
+("Bikes","mountain",27.00, "Co-op_Cycles", "co-op_cycles.png"),
+("Bikes","road",99.00, "Salsa", "salsaa.png"),
+("Bikes","gravel",99.00, "Salsa", "salsab.png"),
+("Bikes","recumbent",299.00, "PERFORMER", "performer.png"),
+("Bikes","tandem",249.00, "Salsa", "salsac.png"),
+("Camping","tent - 1 person",33.00, "Big Agnes", "bigagnes.png"),
+("Camping","tent - 2 person",18.00, "TheNorthFace", "thenorthface.png"),
+("Camping","tent - 3 person",17.00, "Klymit", "klymit.png"),
+("Camping","tent - 4 person",18.00, "Eureka", "eureka.png"),
+("Camping","tent - 6 person",11.00, "Alpine Mountain Gear", "alpinemountaingear.png"),
+("Camping","bear canister - S",7.00, "BearVault", "bearvaulta.png"),
+("Camping","bear canister - M",8.00, "BearVault", "bearvaultb.png"),
+("Camping","bear canister - L",8.00, "BearVault", "bearvaultc.png"),
+("Camping","bear canister - XL",9.00, "BearVault", "bearvaultd.png"),
+("Camping","daypack",6.00, "Osprey", "osprey.png"),
+("Camping","overnight pack",8.00, "Co-op Traverse", "co-optraverse.png"),
+("Camping","backpacking pack",23.00, "Granite Gear", "granitegear.png"),
+("Water sports","canoe",99.00, "Old Town", "oldtown.png"),
+("Water sports","kayak - single",129.00, "EddyLine", "eddyline.png"),
+("Water sports","kayak - tandem",59.00, "TAHE", "tahe.png");
 
 CREATE TABLE "customers" (
```

Comment:
What about the password? And Tim is good with not having a username. A simple way to do that would be to use the username as the password.